opencv/src/OpenCV/ImgProc/GeometricImgTransform.hsc

```diff
@@ -451,27 +457,24 @@ logPolar src center magnitudeScale interpolationMethod inverse fillOutliers =
 <http://docs.opencv.org/3.0-last-rst/modules/imgproc/doc/geometric_transformations.html#getperspectivetransform OpenCV Sphinx doc>
 -}
 getPerspectiveTransform
-    :: (IsPoint2 point2 CFloat, MonadError CvException m)
-    => V.Vector (point2 CFloat) -- ^ Array of 4 floating-point Points representing 4 vertices in source image
-    -> V.Vector (point2 CFloat) -- ^ Array of 4 floating-point Points representing 4 vertices in destination image
+    :: forall m point2. (IsPoint2 point2 CFloat, MonadError CvException m)
+    => V4 (point2 CFloat) -- ^ Array of 4 floating-point Points representing 4 vertices in source image
```

Comment:
Comments could be simplified from 
`Array of 4 floating-point Points representing 4 vertices in source image` to `Vertices in source image`